packages/redux/src/raffles/actions/factories/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Raffles actions factories.
+ */
+
+export { default as fetchRafflesFactory } from './fetchRafflesFactory';
+export { default as fetchRaffleFactory } from './fetchRaffleFactory';
+export { default as fetchRaffleEstimationFactory } from './fetchRaffleEstimationFactory';
+export { default as fetchRaffleParticipationsFactory } from './fetchRaffleParticipationFactory';
```

Comment:
```suggestion
export { default as fetchRaffleParticipationFactory } from './fetchRaffleParticipationFactory';
```
The same for the create and update factories